include/gatekeeper_net.h

```diff
@@ -73,6 +74,10 @@ struct gatekeeper_if {
 	 */
 	rte_atomic16_t	rx_queue_id;
 	rte_atomic16_t	tx_queue_id;
+
+	/* Specify the IPv4 and IPv6 addresses of this interface. */
+	struct in_addr  ip4_addr;
+	struct in6_addr ip6_addr;
```

Comment:
The code is assuming that both addresses are always there. While one address must always be there, there may not be the second address. If you don't want to address this issue in this patch, it's okay, but you must to add the XXX tag here.